app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -52,10 +52,36 @@ public Result emailSignIn() {
         CriteriaContext context = getCriteriaContext(study.getStudyIdentifier());
         
         UserSession session = authenticationService.emailSignIn(context, signInRequest);
-        
+        logAuthenticationSuccess(session);
+
```

Comment:
Is it worth adding some clauses to the emailSignIn() unit test to make sure we're writing metrics, setting cookie, caching request info, etc? To make it easier, maybe a helper method in AuthenticationControllerMockTest verifyCommonSignInFunctionality()?

Similarly for phoneSignIn, reauth, and regular signIn.

Reply:
Added verifyCommonLoggingForSignIns() for these methods.